pkg/pxc/queries/queries.go

```diff
@@ -52,6 +66,100 @@ func New(client client.Client, namespace, secretName, user, host string, port in
 	}, nil
 }
 
+func (p *Database) DisableLogBin() error {
+	_, err := p.db.Exec(`SET @@SESSION.SQL_LOG_BIN = off;`)
+	return errors.Wrap(err, "disable bin logging")
+}
+
+func (p *Database) IsReplica() (bool, error) {
+	rows, err := p.db.Query("show replica status")
+	if err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
+			return false, nil
+		}
+		return false, errors.Wrap(err, "failed to check if pod is replica")
+	}
+	defer rows.Close()
+	cols, err := rows.Columns()
+	if err != nil {
+		return false, errors.Wrap(err, "get columns")
+	}
+	vals := make([]interface{}, len(cols))
+	for i := range cols {
+		vals[i] = new(sql.RawBytes)
+	}
+	for rows.Next() {
+		err = rows.Scan(vals...)
+		if err != nil {
+			return false, errors.Wrap(err, "scan replication status")
+		}
+	}
+	return true, err
+}
+
+func (p *Database) StopAllReplication() error {
+	_, err := p.db.Exec("stop replica")
```

Comment:
feel free to ignore this comment, in general SQL has its own code style. 
variables are lowercased and syntax is uppercased.
you can always check in mysql docs which is uppercase and which is not.
e.g. https://dev.mysql.com/doc/refman/8.0/en/stop-replica.html

Reply:
0c293ac